app.js

```diff
@@ -408,7 +463,7 @@ class JobResult extends LitElement {
 
       .column.controls {
         display: flex;
-        flex-direction: row-reverse;
+        flex-direction: row;
```

Comment:
This was just for the pending state to align the delete icon on the right, but maybe more confusing that way..
Can also add a different style so its stays to the right.

Reply:
Yeah, this is unfortunately quite tough for keyboard or non-visual users, most specifically because when you press the preview button, you have to tab through several more buttons before you find the preview it toggled. The pattern for buttons like this is, that the content they expand is immediately in the tab order after the button. Think... like an UI "accordion". 

I didn't work with the pending state; if that was right-aligned before, I'd expect it to still be. I just reversed the DOM order and removed the `-reverse` CSS directive: it SHOULD look the same. I'll check, and if it doesn't, can add something else to make sure delete comes first and is right-aligned visually when the capture is pending.

Reply:
Fixed 
![image](https://user-images.githubusercontent.com/11020492/92135828-43d5e980-edd9-11ea-86bb-03ee011ff773.png)